PyInstaller/building/makespec.py

```diff
@@ -200,6 +200,14 @@ def __add_options(parser):
                          "  executable, store them as files in the resulting\n"
                          "  output directory.\n"
                          "\n"))
+    g.add_argument("--bootloader-ignore-signals", action="store_true",
+                   default=False,
+                   help=("Tell the bootloader to ignore signals rather "
+                         "than forwarding them to the child process. "
+                         "Useful in situations where e.g. a supervisor "
+                         "process signals both the bootloader and child "
+                         "(e.g. via a process group) to avoid signalling "
+                         "the child twice."))
```

Comment:
What do you think about moving this below `--noupx`, or even into the "Rarely used special options:" section?

Reply:
Sure; though I'm not really sure how common this use case is, I lean towards the latter.